spec/event_sourcery/aggregate_root_spec.rb

```diff
@@ -87,60 +87,88 @@ def initialize(id,
   context 'when state changes' do
     let(:events) { [] }
 
-    subject(:aggregate) {
-      new_aggregate(aggregate_uuid) do
-        def add_item(item)
-          apply_event ItemAdded, body: { id: item.id }
+    context 'and the event is valid' do
```

Comment:
This diff looks very misleading. The only thing I changed from 90 through to 130 was to introduce a context (hence whitespace change) and fix the errant apostrophes.